ui/raidboss/data/07-dt/trial/queen-eternal-ex.txt

```diff
@@ -0,0 +1,211 @@
+88.8 "Legitimate Force" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
+91.9 "Wind of Change" #Ability { id: "A024", source: "Queen Eternal" }
+99.8 "World Shatter" Ability { id: "A01C", source: "Queen Eternal" }
+# End wind miniphase
+
+111.8 "Prosecution of War" Ability { id: "A00A", source: "Queen Eternal" }
+114.9 "Prosecution of War" Ability { id: "A00B", source: "Queen Eternal" }
+125.6 "Divide and Conquer x8" Ability { id: "A017", source: "Queen Eternal" } duration 7
+136.6 "Divide and Conquer" Ability { id: "7729", source: "Queen Eternal" }
+144.6 "Royal Domain" Ability { id: "A04E", source: "Queen Eternal" }
+157.7 "Virtual Shift" Ability { id: "A01A", source: "Queen Eternal" }
+
+# Earth miniphase
+166.8 "Laws of Earth" Ability { id: "A028", source: "Queen Eternal" }
+177.9 "Legitimate Force" Ability { id: ["A01E", "A020"], source: "Queen Eternal" }
+181.0 "Legitimate Force" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
+186.0 "Burst" Ability { id: "A029", source: "Queen Eternal" }
+198.1 "Gravitational Empire" Ability { id: "A02B", source: "Queen Eternal" }
+199.1 "Gravity Pillar" Ability { id: "A02C", source: "Queen Eternal" }
+199.1 "Gravity Ray" Ability { id: "A02D", source: "Queen Eternal" }
+199.9 "Burst" Ability { id: "A029", source: "Queen Eternal" }
+204.2 "Meteor Impact" Ability { id: "A02E", source: "Queen Eternal" }
+211.3 "Meteor Impact" Ability { id: "A02E", source: "Queen Eternal" }
+225.4 "Weighty Blow (cast)" Ability { id: "A033", source: "Queen Eternal" }
+226.0 "Weighty Blow (orb)" Ability { id: "A032", source: "Virtual Boulder" }
+229.1 "Weighty Blow (orb)" Ability { id: "A032", source: "Virtual Boulder" }
+232.2 "Weighty Blow (orb)" Ability { id: "A032", source: "Virtual Boulder" }
+235.3 "Weighty Blow (orb)" Ability { id: "A032", source: "Virtual Boulder" }
+240.5 "World Shatter" Ability { id: "A01C", source: "Queen Eternal" }
+# End earth miniphase
+
+252.6 "Prosecution of War" Ability { id: "A00A", source: "Queen Eternal" }
+255.7 "Prosecution of War" Ability { id: "A00B", source: "Queen Eternal" }
+261.7 "Coronation" Ability { id: "A013", source: "Queen Eternal" }
+267.8 "Atomic Ray (cast)" Ability { id: "A015", source: "Queen Eternal" }
+273.8 "Ruthless Regalia" Ability { id: "A014", source: "Queen Eternal" }
+274.9 "Atomic Ray (damage)" Ability { id: "A016", source: "Queen Eternal" }
+291.0 "Absolute Authority" Ability { id: "A041", source: "Queen Eternal" } duration 13.4
+300.9 "Absolute Authority (Flares/Stack)" Ability { id: "A046", source: "Queen Eternal" }
+305.0 "Absolute Authority (Dorito Stack)" Ability { id: "A048", source: "Queen Eternal" }
+312.1 "Absolute Authority (Knockback)" Ability { id: "A045", source: "Queen Eternal" }
+329.2 "Virtual Shift" Ability { id: "A01B", source: "Queen Eternal" }
+
+# Ice miniphase
+338.3 "Laws of Ice" Ability { id: "A035", source: "Queen Eternal" }
+339.3 "Laws of Ice" Ability { id: "A036", source: "Queen Eternal" }
```

Comment:
```suggestion
338.3 "Laws of Ice" Ability { id: "A035", source: "Queen Eternal" } window 338.3,10
339.3 "Laws of Ice" Ability { id: "A036", source: "Queen Eternal" }
```
Also, one of these should probably be a `--sync--`, unless both are damage instances.

Reply:
Both of these are semi-important. I updated the lines to:

```
339.8 "Laws of Ice (motion)" Ability { id: "A035", source: "Queen Eternal" }
340.8 "Laws of Ice (platforms)" Ability { id: "A036", source: "Queen Eternal" }
```

which hopefully makes more sense?

Reply:
Sounds good to me.